packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -691,6 +706,14 @@ export default defineComponent({
   }
 }
 
+#clipboard-btns.disabled {
```

Comment:
Not a fan that we need this, but love the attention to detail!

Reply:
Yeah I know... the recent outline-change is just ugly when it comes to button groups. May need to re-think that in the future.